nifi-nar-bundles/nifi-email-bundle/nifi-email-processors/src/main/java/org/apache/nifi/processors/email/ListenSMTP.java

```diff
@@ -106,26 +69,17 @@
             .addValidator(StandardValidators.PORT_VALIDATOR)
             .build();
 
-    protected static final PropertyDescriptor SMTP_HOSTNAME = new PropertyDescriptor.Builder()
-            .name("SMTP_HOSTNAME")
-            .displayName("SMTP hostname")
-            .description("The hostname to be embedded into the banner displayed when an " +
-                    "SMTP client connects to the processor TCP port .")
-            .required(true)
-            .expressionLanguageSupported(false)
-            .addValidator(StandardValidators.NON_EMPTY_VALIDATOR)
-            .build();
-
-    protected static final PropertyDescriptor SMTP_MAXIMUM_CONNECTIONS = new PropertyDescriptor.Builder()
+    static final PropertyDescriptor SMTP_MAXIMUM_CONNECTIONS = new PropertyDescriptor.Builder()
             .name("SMTP_MAXIMUM_CONNECTIONS")
             .displayName("Maximum number of SMTP connection")
             .description("The maximum number of simultaneous SMTP connections.")
             .required(true)
+            .defaultValue("1")
             .expressionLanguageSupported(false)
             .addValidator(StandardValidators.INTEGER_VALIDATOR)
             .build();
 
-    protected static final PropertyDescriptor SMTP_TIMEOUT = new PropertyDescriptor.Builder()
+    static final PropertyDescriptor SMTP_TIMEOUT = new PropertyDescriptor.Builder()
```

Comment:
Is SMTP_TIMEOUT being enforced at the DATA stage?

The server seems to hang around the data stage with the connection open no matter how long the client idles for.

Typing . and quit do not work

```
$ time telnet 0 2525
Trying 0.0.0.0...
Connected to 0.
Escape character is '^]'.
220 localhost ESMTP Apache NiFi
helo .
250 localhost
mail from: x
250 Ok
rcpt to: a
250 Ok
data
354 End data with <CR><LF>.<CR><LF>
Hmmmm....

.
^]
telnet> quit
Connection closed.

real    1m24.348s
user    0m0.000s
sys     0m0.002s
```